src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/CompilationErrorPage.cshtml

```diff
@@ -34,6 +34,10 @@
                         <div class="titleerror">@fileName</div>
                     }
                 }
+                @if (!string.IsNullOrEmpty(errorDetail.ErrorMessage))
+                {
+                    <details>@errorDetail.ErrorMessage</details>
```

Comment:
@Eilon doesn't this create a control that's currently closed but can be opened to show the details? `open="open"` would create a control that's open but can be closed.

@pranavkm which browsers support `<details>`?